helm/presets/aws.yaml

```diff
@@ -101,6 +104,14 @@ presto:
 cert_manager:
   enabled: true
 
+global:
```

Comment:
Do you need to add this.  I think this reverts a change jeff recently made?

Reply:
I'm guessing we just messed up the merge